Makefile

```diff
@@ -234,3 +234,6 @@ shake_concourse_volumes: check-env ## Restarts concourse services and workers an
 show-cf-memory-usage: ## Show the memory usage of the current CF cluster
 	$(eval export API_ENDPOINT=https://api.${SYSTEM_DNS_ZONE_NAME})
 	@./scripts/show-cf-memory-usage.rb
+
+fix_sso_url: check-env
```

Comment:
This `check-env` makes it a bit annoying to use because of the AWS credentials check - you don't need AWS credentials to run this and it slows down the execution by a second or tow.